presto-iceberg/src/test/java/com/facebook/presto/iceberg/procedure/TestExpireSnapshotProcedure.java

```diff
@@ -200,7 +200,7 @@ public void testExpireSnapshotUsingNamedArgsOfSnapshotIds()
         Session session = getSession();
         String tableName = "named_args_snapshot_ids_table";
         try {
-            assertUpdate(session, "CREATE TABLE " + tableName + " (id integer, data varchar)");
+            assertUpdate(session, "CREATE TABLE IF NOT EXISTS " + tableName + " (id integer, data varchar)");
```

Comment:
Do we need these changes in `TestExpireSnapshotProcedure`? All tests in this class operate on tables with different table names, so it seems unlikely to meet the same failures as `TestRollbackToTimestampProcedure`. Any misunderstanding or overlook please let me know.

Reply:
Apologies for the confusion, this is not related to failure. Just made this change to made consistency in the tests